src/com/company/OrderProcessor.java

```diff
@@ -7,10 +7,18 @@
 import java.time.format.DateTimeFormatter;
```

Comment:
ogolnie rozwiazanie w porzadku, ale jesli chcemy wejsc na wyzszy poziom to w kodzie produkcyjnym na projekcie takie cos raczej byloby rozwiazane przy pomocy tych nowszych bardziej zaawansowanych narzedzi, jesli masz czas i ochote to mozesz sprobowac pobawic sie z ExecutorService:

- Utworzenie puli wątków (np. ExecutorService executor = Executors.newFixedThreadPool(4);)
- Dodawanie nowych zamówień do kolejki zadań (executor.submit(() -> new OrderProcessor(order).run()))
- W ten sposób łatwiej skalować liczbę wątków. Gdy zamówień będzie bardzo dużo, nie zalewasz systemu setkami wątków, tylko pracujesz w stabilnej puli, która jest wielokrotnie wykorzystywana.